3ds/source/gui/overlay/MoveOverlay.cpp

```diff
@@ -66,11 +66,14 @@ MoveOverlay::MoveOverlay(ReplaceableScreen& screen, const std::variant<std::shar
 {
     instructions.addBox(false, 75, 30, 170, 23, COLOR_GREY, i18n::localize("SEARCH"), COLOR_WHITE);
     const std::vector<std::string>& rawMoves = i18n::rawMoves(Configuration::getInstance().language());
-    for (int i = 1; i <= TitleLoader::save->maxMove(); i++)
+    const std::set<int>& availableMoves      = TitleLoader::save->availableMoves();
+    for (auto i = availableMoves.begin(); i != availableMoves.end(); i++)
     {
-        if (i >= 622 && i <= 658)
+        if (*i >= 622 && *i <= 658)
```

Comment:
Could combine the if statements

Reply:
Actually decided to handle this in a different way; I added a case in the if to sort 0 forwards